tests/tape/tapes/test_qnode.py

```diff
@@ -25,8 +25,13 @@ class TestValidation:
     def test_invalid_interface(self):
         """Test that an exception is raised for an invalid interface"""
         dev = qml.device("default.qubit", wires=1)
+        test_interface = "something"
+        expected_error = (
+            fr"Unknown interface {test_interface}\. Interface must be "
+            r"one of \['autograd', 'torch', 'tf'\]\."
+        )
 
-        with pytest.raises(qml.QuantumFunctionError, match="Unknown interface"):
+        with pytest.raises(qml.QuantumFunctionError, match=expected_error):

```

Comment:
A case where the bug was mixed because we were not matching the _full_ exception message!